snapshot/lib/snapshot/simulator_launchers/simulator_launcher_xcode_8.rb

```diff
@@ -101,6 +102,35 @@ def execute(command: nil, language: nil, locale: nil, device_type: nil, launch_a
                                                   UI.crash!("Too many errors... no more retries...")
                                                 end
                                               end)
+      stop_recording
+    end
+
+    def start_recording(dir_name, device_type, name)
```

Comment:
if this is identical to above, can this be moved outside of these files somehow and unified?